akka-stream-tests/src/test/scala/akka/stream/scaladsl/AttributesSpec.scala

```diff
@@ -60,6 +60,17 @@ class AttributesSpec extends AkkaSpec with ScalaFutures {
         attributes.get[Name] should contain(Name("attributesSink"))
       }
     }
+
+    val attributes = Attributes.name("a") and Attributes.name("b") and Attributes.inputBuffer(1, 2)
```

Comment:
Seeing `Name("attributesSink")` above I think you may be able to drop the `Attributes.` part.